cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,8 @@ class ProductsListPage {
   private checkoutBtn: string;
 
   constructor() {
-    this.addCarBtn = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.checkoutBtn = "[style*='display: block;'] .button-container > a";
+    this.addCarBtn = '#center_column a.button.ajax_add_to_cart_button.btn.btn-default';
```

Comment:
here you can usse an attribute to indicate we are looking for the add to cart button

Reply:
resolved